framework/include/utils/RankFourTensorImplementation.h

```diff
@@ -383,6 +399,22 @@ RankFourTensorTempl<Real>::invSymm() const
   return result;
 }
 
+template <typename T>
```

Comment:
@dschwen: This now fails due to `EIGEN_STACK_ALLOCATION_LIMIT ` on the new "size 250" AD test.  I'm inverting a full Rank 3 here (so a 9x9 matrix) and apparently the size of the `MetaPhysicL::DualNumber` x 81 exceeds the default Eigen stack limit of 128 kB.

You can increase the Eigen stack limit with `EIGEN_STACK_ALLOCATION_LIMIT` but that either needs to happen as a `-D` during compile or a `#define` before the first Eigen header gets pulled in, which unfortunately I think is in libmesh.  I think increasing this limit is probably the right thing to do, as you don't want this going on the heap.

Let me know what you suggest please.

Reply:
I reverted to LAPACK, which should fix the issue for now.  Long-term there's no problem with `Eigen` apart from the stack limit for large `MetaPhysicL::DualNumber` sizes.